util/Migrator/DbMigrator.cs

```diff
@@ -23,10 +23,40 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
         }.ConnectionString;
     }
 
-    public bool MigrateMsSqlDatabase(bool enableLogging = true,
+    public bool MigrateMsSqlDatabaseWithRetries(bool enableLogging = true,
         CancellationToken cancellationToken = default(CancellationToken))
     {
-        if (enableLogging && _logger != null)
+        var attempt = 1;
+
+        while (attempt < 10)
+        {
+            try
+            {
+                var success = MigrateDatabase(enableLogging, cancellationToken);
+                return success;
+            }
+            catch (SqlException e)
+            {
+                if (e.Message.Contains("Server is in script upgrade mode") && attempt < 10)
+                {
+                    attempt++;
+                    _logger.LogInformation("Database is in script upgrade mode. " +
+                        "Trying again (attempt #{0})...", attempt);
+                    Thread.Sleep(20000);
```

Comment:
ℹ️ Per my other comment, this just seemed a bit arbitrary to me but it has its reasons I suppose, and it's also five years old via https://github.com/bitwarden/server/commit/6f1f2305e352bf9fd5a4e5ed484254a666737674.

Reply:
I also don't know why the value is 20 seconds, but I followed the code that you pointed to.